src/rt/dwarfeh.d

```diff
@@ -746,9 +746,9 @@ LsdaResult scanLSDA(const(ubyte)* lsda, _Unwind_Ptr ip, _Unwind_Exception_Class
  *      exceptionClass = which language threw the exception
  *      lsda = pointer to LSDA table
  * Returns:
- *      >=1 means the handler index of the classType
- *      0 means classType is not in the Action Table
- *      <0 means corrupt
+ *      - &gt;=1 means the handler index of the classType
+ *      - 0 means classType is not in the Action Table
+ *      - &lt;0 means corrupt
```

Comment:
I wonder why was this necessary, as both Ddoc and Ddox are supposed to escape this Sunday automatically.

Reply:
The > was interpreted as a block quote in markdown. I added the dashes to make it into a list as the glued together statements looked bad.

Reply:
This is what it looked like before the change: http://dtest.dlang.io/artifact/website-36c76465537501a0b34f32792f5a76f1f9b3888d-01a9ae3ff87a8f9c5eb4438e29e16951/web/phobos-prerelease/rt_dwarfeh.html#.actionTableLookup

Reply:
Ah yes, thanks for checking